apps/skilavottord/ws/src/app/modules/accessControl/accessControl.service.ts

```diff
@@ -13,61 +14,60 @@ import {
 
 @Injectable()
 export class AccessControlService {
-  constructor(@Inject(LOGGER_PROVIDER) private logger: Logger) {}
+  constructor(
+    @InjectModel(AccessControlModel)
+    private accessControlModel: typeof AccessControlModel,
+  ) {}
 
-  async findAll(): Promise<AccessControlModel[]> {
-    this.logger.info('---- Starting findAll Access Controls ----')
+  async findAll(isDeveloper: boolean): Promise<AccessControlModel[]> {
+    return this.accessControlModel.findAll({
+      where: isDeveloper ? {} : { role: { [Op.not]: Role.developer } },
+      include: [
+        {
+          model: RecyclingPartnerModel,
+        },
+      ],
+    })
+  }
 
-    // TODO replace mock data with actual db query
-    const res = await Promise.resolve([
-      {
-        nationalId: '1234567777',
-        name: 'Gervimaður',
-        role: Role.recyclingCompany,
-        partnerId: '8888888888',
-      },
-      {
-        nationalId: '1234567888',
-        name: 'Gervimaður2',
-        role: Role.recyclingCompany,
-        partnerId: '9999999999',
-      },
-      {
-        nationalId: '1234567899',
-        name: 'Gervimaður3',
-        role: Role.recyclingCompany,
-        partnerId: '9999999999',
-      },
-    ] as AccessControlModel[])
-    return res
+  async findOne(nationalId: string): Promise<AccessControlModel> {
+    // TODO check this out
```

Comment:
Need to figure out when this happens

Reply:
it was because of raw: true, adding it solved the issue, this is fixed